vim_onenote.ahk

```diff
@@ -117,77 +216,77 @@ Send, {End}{Enter}
 Gosub InsertMode
 return
 
+r::
+    send +{right}
+    gosub, InsertMode
+    ; Wait for single key to be pressed, sends that key and returns to normal
+    input, inp, V E L1
+    send {left}
+    gosub, NormalMode
+return
+
 ; undo
 u::Send, ^z
-return 
 ; redo.
 ^r::Send, ^y
-return 
 
-+G:: Send, ^{End}
 ; G goto to end of document
-return
+sG(){
```

Comment:
I understand this. But in this case, the name of the method (and its consistency) is a vital part of the code in terms of extensibility. +g goes to end of doc. I name the function it calls shiftG. A generic motion is requested, and the character returned is G. Code sees this is an upper case letter, so it was sent with shift+g. Knows that the relevant function to call is therefore shiftG.
This would also work with any other motion command that relies on a shift, without adding extra code to handle it.
Does that make sense?